new/PRS13297Zamalakot.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13297Zamalakot" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Zamalakot</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="ABr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="ABr" when="2020-12-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘመለኮት</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamalakot</persName>
+                    <faith type="EOTC"/>
+                    <floruit notBefore="1600" notAfter="1800"></floruit>
+
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13297Zamalakot" passive="GCA001">
+                    <desc>He is mentioned in the colophon of the manuscript GCA001 as commissioner of the manuscript and donor to the church of Faqādā</desc>
```

Comment:
In the desc, GCA001 can be marked up this way: `<ref type="mss" corresp="GCA001"/>`
If Faqādā has its own ID, also this word can be marked up as a placeName: `<placeName ref="..."/>`

Reply:
> In the desc, GCA001 can be marked up this way: `<ref type="mss" corresp="GCA001"/>`

Alright! Thank you, Massimo

> If Faqādā has its own ID, also this word can be marked up as a placeName: `<placeName ref="..."/>`

There are two placeName called Faqādā in BM, recorded by EthioSpare. I'm checking in the single manuscripts preserved in these monasteries if some of the names found in our ms are attested also there. If not, I will create a new record for the place Faqādā. I will come back to you when I'm done with the search.

Reply:
Sorry, I have to correct what I wrote before. There is only one church in BM (Faqādā Māryām), the other is a location. Anyway, I found in three mss of Faqādā Māryām (FBM-001, FBM-003, FBM-016) the name Baʾǝda Māryām attested as owner. In our ms (GCA001) is also present in the subscriptiones and blessing formulae a Baʾǝda Māryām and the ms. was donated to a መቅደሰ፡ ፈቃዳ፡. Do you think our  መቅደሰ፡ ፈቃዳ፡ can be identified with Faqādā Māryām recorded by Ethiospare?

Reply:
Dear Anto, as far as I remember correctly, this was exactly what I also assumed when preparing our inventory. We also stated Faqādā as origin/provenance in our inventory. I have to check in the office tomoroow wehther if I have more details on it.

Reply:
Thank you, Sanne, for that. I remembered something about a known place recurring also in our manuscripts but I did not linked to Faqādā. Alright, I will wait that you check before I proceed with creating a new record both for Faqādā and Baʾǝda Māryām.

P.S. Tomorrow morning we have our breakfast at 9.30. Massimo, if you want to join us you are more than welcome!

Reply:
You could add also here that this person has a relation to Faqādā Māryām


```
<affiliation>
<placeName ref="INS0071FBM">Faqādā Māryām</placeName>
</affiliation>
```




I have never done this but checked in the TEI guidelines: https://tei-c.org/release/doc/tei-p5-doc/de/html/ref-affiliation.html